app/web/features/profile/view/NewHostRequest.tsx

```diff
@@ -81,6 +73,7 @@ export default function NewHostRequest({
   const isPostBetaEnabled = process.env.NEXT_PUBLIC_IS_POST_BETA_ENABLED;
   const [numVisitors, setNumVisitors] = useState(1);
   const user = useProfileUser();
+  const { t } = useTranslation([GLOBAL]);
 
```

Comment:
Even though this component is in the `profile` folder... it's debatable whether it should be here since it's more about sending host requests. Maybe these messages/constants should be in `messages` instead